marketplacetests/tests/test_marketplace_incorrect_pin.py

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from fxapom.fxapom import FxATestAccount
+
+from marketplacetests.marketplace_gaia_test import MarketplaceGaiaTestCase
+from marketplacetests.marketplace.app import Marketplace
+from marketplacetests.payment.app import Payment
+
+
+class TestMarketplaceIncorrectPin(MarketplaceGaiaTestCase):
+
+    def test_incorrect_pin(self):
+
+        app_name = 'Test Zippy With Me'
+        valid_pin = '1234'
+        invalid_pin = '1111'
+        acct = FxATestAccount(base_url=self.base_url).create_account()
+
+        if self.apps.is_app_installed(app_name):
+            self.apps.uninstall(app_name)
+
+        marketplace = Marketplace(self.marionette, self.MARKETPLACE_DEV_NAME)
+        marketplace.launch()
+
+        marketplace.login(acct.email, acct.password)
+
+        marketplace.set_region('United States')
+
+        details_page = marketplace.navigate_to_app(app_name)
```

Comment:
That's a good point, and while that extra step may be needed for one of the tests, we surely don't need it for all of the tests that are just meant to test payment functions. I will open an issue about that.